### ISSUE TITLE
2022-1-5

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#702]
120P94, MAML 2; define a set of learning algorithm; network structure & init & update让机器自己来学, 之前是人为制定的超参数; meta learning就是学习超参数; 需要很多任务来评估; (2021-12-25)
121P95, MAML 3; training tasks (training examples & testing examples); testing tasks (training examples & testing examples); few-shot是目的, meta是手段; L(F) = sum(l); (2021-12-25)
122P96, MAML 4; Omniglot; N-way K-shot, N classes, K examples; (2021-12-25)
123P97, MAML 5; MAML; 学习初始化参数init, 限制network structure必须一样; MAML(loss计算的是训练以后的表现) difference between pre-train (loss计算的是现在的表现); train update once, test update multiple times; 为了节省计算资源, 为了防止over-fitting; (2021-12-25)
124P98, MAML 6; toy example; (2021-12-25)
125P99, MAML 7; warning of math; 假装没有看到二次微分; (2021-12-26)
126P100, MAML 8; implementation of MAML; translation; (2021-12-27)
127P101, MAML 9; Reptile; 朝四步以后的方向走一步; 介于pre-traini和MAML之间; learn network structure OR learning rate; output parameter directly? (2021-12-27)
128P102, metric-based 1; face verification; meta learning; siamese network; (2021-12-27)
129P103, metric-based 2; binary classification; 不同角度的同一张人脸投影到相似位置; learn to ignore the background; (2021-12-27)
